### PR TITLE
Add callstats support for mobile

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

**Comment by saghul** [comment]
I set a WIP label because I have only tested with a debug build callstats provided me with, I'll update once we have the definitive build.

Note that despite the naming, react-native-callstats does not depend on react-native. I'll try to convince them to rename the package.

File: modules/statistics/statistics.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import JitsiTrackError from '../../JitsiTrackError';
 const logger = require('jitsi-meet-logger').getLogger(__filename);
 const LocalStats = require('./LocalStatsCollector.js');
 const RTPStats = require('./RTPStatsCollector.js');
+
+import RTCBrowserType from '../RTC/RTCBrowserType';
```
Comment (paweldomas) [inline modules/statistics/statistics.js]: is that in order ? shouldn't it come before RTPStats ?

File: modules/statistics/CallStats.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
-/* global $, callstats */
+/* global callstats */
 const logger = require('jitsi-meet-logger').getLogger(__filename);
-const GlobalOnErrorHandler = require('../util/GlobalOnErrorHandler');
 
-const jsSHA = require('jssha');
-const io = require('socket.io-client');
+import RTCBrowserType from '../RTC/RTCBrowserType';
+import GlobalOnErrorHandler from '../util/GlobalOnErrorHandler';
```
Comment (paweldomas) [inline modules/statistics/CallStats.js]: not in order ?

I'm not sure whether we order by class names of file names ?

File: modules/statistics/statistics.js [hunk 3]
```diff
@@ -104,7 +106,7 @@ function Statistics(xmpp, options) {
             // of callstats.io may be disabled because of globally-disallowed
             // requests to any third parties.
             && (Statistics.disableThirdPartyRequests !== true);
-    if (this.callStatsIntegrationEnabled) {
+    if (this.callStatsIntegrationEnabled && !RTCBrowserType.isReactNative()) {
```
Comment (paweldomas) [inline modules/statistics/statistics.js]: if we skip this if for RN then the check at 112 should be moved outside

Comment (paweldomas) [inline modules/statistics/statistics.js]: or only the loading part should be under isReactNative condition

Comment (saghul) [inline modules/statistics/statistics.js]: oh, I see. That wasn't there before so I missed it during the rebase. Will fix.

File: modules/statistics/statistics.js (hunk 1 above)
Comment (saghul) [inline modules/statistics/statistics.js]: Right you are sir!

File: modules/statistics/CallStats.js (hunk 2 above)
Comment (saghul) [inline modules/statistics/CallStats.js]: file names

**Comment by saghul** [comment]
@paweldomas Thanks for the review! I think I addressed all your comments, can you PTAL?

**Pull request merged**